eng/common/scripts/Detect-Api-Changes.ps1

```diff
@@ -55,11 +55,20 @@ function Submit-Request($filePath, $packageName)
     }
     $uri = [System.UriBuilder]$APIViewUri
     $uri.query = $query.toString()
+
+    $correlationId = [System.Guid]::NewGuid().ToString()
```

Comment:
What is this needed for?

Reply:
It is added to the request and all the telemetry data. Make it easy to locate all related logging and telemetry, even when we have cases like a timeout.